File: src/main/java/com/kaiosantiago/laudopro/dtos/user/UserDto.java
package com.kaiosantiago.laudopro.dtos.user;

import com.kaiosantiago.laudopro.dtos.workOrder.WorkOrderDto;
import com.kaiosantiago.laudopro.entity.Plan;
import com.kaiosantiago.laudopro.entity.Role;
import lombok.Data;

import java.util.List;

@Data
public class UserDto {
    private String uuid;
    private String name;
    private String email;
    private Long planId;
    private Long roleId;
    private List<WorkOrderDto> workOrders;
}

File: DTOs/UserDto.cs
namespace LaudoPro.DTOs
{
    public class UserDto
    {
        public string Uuid { get; set; }
        public string Name { get; set; }
        public string Email { get; set; }
        public string Password { get; set; }
        public int PlanId { get; set; }
        public int RoleId { get; set; }
    }
}
